src/display/MovablePointDisplay.test.ts

```diff
@@ -0,0 +1,7 @@
+import { MovablePointDisplay } from "./MovablePointDisplay"
+
+describe("MovablePointDisplay", () => {
+  it("has a human-readable displayName", () => {
+    expect(MovablePointDisplay.displayName).toBe("MovablePointDisplay")
+  })
+})
```

Comment:
I like these, but I feel like it'd be better to just consolidate all displayName expectations into one big test. Not a blocker on this PR.